lib/msal-core/src/telemetry/TelemetryManager.ts

```diff
@@ -55,7 +55,11 @@ export default class TelemetryManager {
             sdk: "msal.js",
             sdkVersion: libraryVersion(),
             applicationName,
-            applicationVersion
+            applicationVersion,
+            networkInformation: {
+                // @ts-ignore
+                effectiveType: navigator && navigator.connection && navigator.connection.effectiveType
```

Comment:
I would maybe call our property something else, like `connectionSpeed`.

Reply:
Also, I would consider adding an enum, as there is a known set of expected values:  'slow-2g', '2g', '3g', or '4g'

Reply:
good stuggestion